### DISCUSSION TITLE
Adding a "Troubleshooting" documentation ?

### DISCUSSION THREAD

**Discussion by agoncal** [microsoft/NubesGen#109] [Ideas]
Hi,

Using NubesGen is also using Azure.... and therefore, facing a few problems if you are new to Azure. Typically, when using NubesGen I got:

```
(1/8) Checking Azure CLI status...
(2/8) Checking GitHub CLI status...
(3/8) Create resource group "rg-terraform-001"
The subscription '7cfe16d7-a086-41c2-9965-77a57954ac47' could not be found. 
```

So I used a few commands to sort it out:

```
az account list
az account show
az account set --subscription mysubscription
```

Do you think this kind of help could go into a `docs/troubleshooting.md` page ? Or I create an issue with the answer so people can get some help from the issue ?

**Comment by jdubois** [comment]
Yes, a docs page with the common issues would be awesome!
It would also be a great place to look for improvement -> ideally this page should be empty, as everything should be taken care of in the tool.

**Reply by agoncal** [reply]
Do you want me to create it with the issue I got ?

**Reply by jdubois** [reply]
Oh yes if you want to -> more contributors is always the best, welcome!

**Reply by agoncal** [reply]
Done : https://github.com/microsoft/NubesGen/pull/113
